Library/Formula/boost.rb

```diff
@@ -36,6 +36,10 @@ class Boost < Formula
     cause "Dropped arguments to functions when linking with boost"
   end
 
+  # Patch for odeint. Can (hopefully) be removed with the next release of Boost (1.58).
+  # See https://github.com/headmyshoulder/odeint-v2/issues/138
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!